x-pack/plugins/remote_clusters_management/public/lib/manage_angular_lifecycle.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { unmountComponentAtNode } from 'react-dom';
+
+export const manageAngularLifecycle = ($scope, $route, elem) => {
```

Comment:
I feel like putting this logic into its own file is indirection without gaining any abstraction or other benefits like easier unit testing. How would you feel about moving it into `index.js` the way it is in rollups?https://github.com/elastic/kibana/blob/master/x-pack/plugins/rollup/public/crud_app/index.js#L61. Personally I prefer this because I can see all of the "React <-> Angular configuration" in one place but I'm open to keeping it as is if you feel differently.

Reply:
I don't feel strongly either way, this was copied from index management 😄I'll play around with the two approaches as I work on fleshing the store out, that'll give me a better feel of which one I prefer, then I'll update this as necessary